src/egon/data/datasets/scenario_path/import_status2019.py

```diff
@@ -0,0 +1,114 @@
+"""
+Read eTraGo tables for the status2019 and import it to db
+"""
+
+from pathlib import Path
+from urllib.request import urlretrieve
+import os
+import subprocess
+
+import pandas as pd
+
+from egon.data import config, db
+from egon.data.datasets import Dataset
+import egon.data.config
+
+
+class Import_Status2019(Dataset):
+    def __init__(self, dependencies):
+        super().__init__(
+            name="import_status2019",
+            version="0.0.1",
+            dependencies=dependencies,
+            tasks=(import_scn_status2019,),
+        )
+
+
+sources = egon.data.config.datasets()["scenario_path"]["sources"]
+
+
+def download_status2019():
+    """
+    Download the status2019 etrago tables from Zenodo
+
+    Returns
+    -------
+    None.
+
+    """
+    # Get parameters from config and set download URL
+    url = sources["url_status2019"]
+    status2019_path = Path(".") / "PoWerD_status2019.backup"
+
+    # Retrieve files
+    urlretrieve(url, status2019_path)
+
+    return
+
+
+def import_scn_status2019():
+    """
+    Read and import the scenario status2019 and import it into db
+
+    Parameters
+    ----------
+    *No parameters required
+
+    """
+    # Connect to the data base
+    con = db.engine()
+
+    # Clean existing data for status2019
+    tables = pd.read_sql(
+        """
+        SELECT tablename FROM pg_catalog.pg_tables
+        WHERE schemaname = 'grid'
+        """,
+        con,
+    )
+
+    tables = tables[
+        ~tables["tablename"].isin(
+            [
+                "egon_etrago_carrier",
+                "egon_etrago_temp_resolution",
+            ]
+        )
+    ]
+
+    for table in tables["tablename"]:
+        db.execute_sql(
+            f"""
+        DELETE FROM grid.{table} WHERE scn_name = 'status2019';
+        """
+        )
+
+    my_env = os.environ.copy()
+    my_env["PGPASSWORD"] = sources["PGPASSWORD"]
+
+    config_data = config.settings()["egon-data"]
+    database = config_data["--database-name"]
+    host = config_data["--database-host"]
+    port = config_data["--database-port"]
+    user = config_data["--database-user"]
```

Comment:
The password is stored in config_data as well, isn't it? At least it is listed in my config files.

Reply:
As you mentioned, a parameter called _database-password_ is in the configuration file. But it can be modified anytime by the user, and then it would not match the fixed password that the status2019 has.

Reply:
But that is the same for all parameters, isn't it? 
Other functions also access the password from there, so it would be a problem anyhow.

Reply:
If that is the case, shouldn't we take _database-password_ out from the configuration file?

Reply:
Change applied in: a7cd357c0f372ca9a1fa3363b82c0cabf773ec66

Reply:
Sorry, I completely forgot to reply. Looks good to me!